astropy/samp/tests/test_client.py

```diff
@@ -38,6 +38,16 @@ def samp_hub(request):
     request.addfinalizer(my_hub.stop)
 
 
+def test_SAMPIntegratedClient_notify_all(samp_hub):
+    """Test that SAMP returns a warning if no receiver got the message."""
+    client = SAMPIntegratedClient()
+    client.connect()
+    message = {"samp.mtype": "coverage.load.moc.fits"}
+    with pytest.raises(SAMPProxyError):
```

Comment:
As someone unfamiliar with SAMP internals, I fail to see how this error can stem from the warning you added. Can you please clarify?

Also, do you need help squashing the commits when this PR is ready?

Thanks!